src/main/java/com/webonise/rbs/controller/RoomController.java

```diff
@@ -0,0 +1,65 @@
+package com.webonise.rbs.controller;
+
+import com.webonise.rbs.entity.Room;
+import com.webonise.rbs.service.RoomServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+@Controller
+@RequestMapping(value = "/room")
+public class RoomController {
+
+    @Autowired
+    RoomServiceImpl roomService;
+
+    @GetMapping(value = "/",produces = MediaType.TEXT_HTML_VALUE)
+    public String savePage(Model model) {
+        model.addAttribute("room", new Room());
+        model.addAttribute("allRooms", roomService.getAllRooms());
+        return "viewRooms";
+    }
+
+    @PostMapping(value = "/save")
+    public String saveRoom(@ModelAttribute("room") Room room, final RedirectAttributes redirectAttributes) {
+        if(roomService.addRoom(room)!=null) {
+            redirectAttributes.addFlashAttribute("saveRoom", "success");
+        } else {
+            redirectAttributes.addFlashAttribute("saveRoom", "failure");
+        }
+        return "redirect:/room/";
+    }
+
+    @GetMapping(value = "/{operation}/{id}")
+    public String editRemoveRoom(@PathVariable("operation") String operation, @PathVariable("id") Long id, final RedirectAttributes redirectAttributes, Model model) {
+        if(operation.equals("delete")) {
```

Comment:
What does this method do actually? It seems that it tries to do 2 task according to operation type. Not recommended.